src/templates/bootstrap/components/htmlelement.html

```diff
@@ -0,0 +1,5 @@
+<div [formGroup]="form" class="form-group">
```

Comment:
You do not need any [formGroup] or class="form-group" declaration here. There would never be any input elements within an html element component.